samples/bluetooth/fast_pair/input_device/README.rst

```diff
@@ -484,7 +484,9 @@ Fast Pair GATT Service
 This sample uses the :ref:`bt_fast_pair_readme` and its dependencies and is configured to meet the requirements of the Fast Pair standard.
 See :ref:`ug_bt_fast_pair` for details about integrating Fast Pair in the |NCS|.
 
-The :ref:`bt_fast_pair_provision_script` is used by the build system to automatically generate the hexadecimal file that contains Fast Pair Model ID and Anti Spoofing Private Key.
+This sample enables both the ``SB_CONFIG_BT_FAST_PAIR`` and :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig options to allow building either with or without Sysbuild.
```

Comment:
This sample should only support sysbuild. Please do not claim that it allows building without it as it would work incorrectly for some targets

Reply:
Ok